packages/hydrated_cubit/lib/src/hydrated_storage.dart

```diff
@@ -62,6 +76,7 @@ class HydratedStorage implements Storage {
       final box = await hive.openBox<dynamic>(
         'hydrated_box',
         encryptionCipher: encryptionCipher,
+        compactionStrategy: compactionStrategy ?? defaultCompactionStrategy,
```

Comment:
In the test with `TODO(need_help)` I actually want to make sure this line is present and does what it should